app/compile/template/index.js

```diff
@@ -118,7 +135,19 @@ export default async function template(files, options = {}) { // eslint-disable-
 
     let { locales: locales_to_build } = locals
     delete locals.locales
-    locales_to_build = to.array(locales_to_build, /[\s,]+/)
+
+    if (locales_to_build.includes('all')) {
+      locales_to_build = all_locales
+    }
```

Comment:
add an else in here that checks to see if all the `locales_to_build` are in `all_locales`. If any of them aren't in `all_locales` then throw an error